config/redis.go

```diff
@@ -31,19 +31,19 @@ type redisConfig struct {
 func init() {
 	config.Redis = redis{
 		Main: redisConfig{
-			Enabled:                            constants.RedisEnabled,
-			Url:                                constants.RedisDefaultUrl,
-			Username:                           constants.RedisDefaultUsername,
-			Password:                           constants.RedisDefaultPassword,
-			OperationMode:                      constants.RedisDefaultOperationMode,
-			MaxActiveConnections:               constants.RedisMaxActiveConnections,
-			MaxIdleConnections:                 constants.RedisMaxIdleConnections,
-			IdleTimeoutInSeconds:               constants.RedisIdleTimeoutInSeconds,
-			CrashAppOnConnectionFailure:        constants.RedisCrashAppOnConnectionFailure,
-			ConnectRetryIntervalInSeconds:      constants.RedisConnectRetryIntervalInSeconds,
-			AutoExpireTopLevelKeysAfterSeconds: constants.RedisAutoExpireTopLevelKeysAfterSeconds,
-			AppNamespace:                       constants.RedisAppNameSpace,
-			Address:                            constants.RedisDefaultAddr,
+			Enabled:                            cache.RedisEnabled,
```

Comment:
I think we would have to remove the config module from ground and put it in the **pine** project later. There is no way that I can see a generic configuration manager being created unless we also enforce some structure around it, which I don't want to do in ground. Pine would be much better for that!